### PR TITLE
Support loopy callables branch

### PR DESCRIPTION
**Pull request by kaushikcfd**

### CODE REVIEW

File: .github/workflows/ci.yml [hunk 1]
```diff
@@ -66,6 +66,18 @@ jobs:
                 curl -L -O -k https://gitlab.tiker.net/inducer/ci-support/raw/master/build-and-test-py-project-within-miniconda.sh
                 . ./build-and-test-py-project-within-miniconda.sh
 
+    pytest-loopycallables:
```
Comment (inducer) [inline .github/workflows/ci.yml:69]: Could you also run the examples against both branches?

Comment (inducer) [inline .github/workflows/ci.yml:69]: Github has this nifty "matrix" build facility,  maybe that can help cut down on repetition. [example](https://github.com/inducer/pytools/blob/637feb3b51849df53c7e15ee1367a82ce90e97d2/.github/workflows/ci.yml#L59-L68)

File: .gitlab-ci.yml [hunk 2]
```diff
@@ -15,6 +15,24 @@ Python 3 POCL:
       junit: test/pytest.xml
 
 
+Python 3 POCL (Loopy callables):
```
Comment (inducer) [inline .gitlab-ci.yml:18]: Examples, too. :)

Comment (inducer) [inline .gitlab-ci.yml:18]: Could you push a version of this to Gitlab (and crosslink PR/MR) to see how they do?

**Review by inducer — COMMENTED** [review]
Thanks for working on this. Some CI comments to start.

File: .github/workflows/ci.yml (hunk 1 above)
Comment (kaushikcfd) [reply .github/workflows/ci.yml]: Added.

Comment (kaushikcfd) [reply .github/workflows/ci.yml]: Done, that was handy! Gitlab has this too! Github: 199c9c5, Gitlab: d2170e5

File: .gitlab-ci.yml (hunk 2 above)
Comment (kaushikcfd) [reply .gitlab-ci.yml]: Related gitlab PR: https://gitlab.tiker.net/inducer/pytato/-/merge_requests/20

Comment (kaushikcfd) [reply .gitlab-ci.yml]: Done.

**Comment by inducer** [comment]
There are some CI fails there. Could you take a look?

**Comment by kaushikcfd** [comment]
Yep, looks like it loopy's kernel callables needs a master merge.

**Comment by kaushikcfd** [comment]
Merged `loopy/master` into `loopy/kernel_callables`. Tests pass now.

**Comment by inducer** [comment]
LGTM. Thanks for working on this.

**Pull request merged**